### PR TITLE
IGAPP-430: Prevent showing of last update before webview content is re…

### PR DESCRIPTION
**Pull request by SteffiStuffel**
…ndered

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/.idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -32,4 +34,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```
Comment (Znippet) [inline native/.idea/codeStyles/Project.xml:37]: Were the changes to the codeStyles intentional?

**Review by Znippet — APPROVED** [review]

File: native/.idea/codeStyles/Project.xml (hunk 1 above)
Comment (SteffiStuffel) [reply native/.idea/codeStyles/Project.xml:37]: Yeah it is. It should prevent the auto-formatting from putting every parameter on a separate line.

File: native/.idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
       <option name="SPACE_BEFORE_GENERATOR_MULT" value="true" />
       <option name="USE_DOUBLE_QUOTES" value="false" />
       <option name="FORCE_QUOTE_STYlE" value="true" />
-      <option name="OBJECT_LITERAL_WRAP" value="2" />
+      <option name="OBJECT_LITERAL_WRAP" value="0" />
```
Comment (klinzo) [inline native/.idea/codeStyles/Project.xml:12]: I think the option we want is `chop down if to long`:
```suggestion
      <option name="OBJECT_LITERAL_WRAP" value="5" />
```
This option keeps all params in one line if shorter than max line length and puts each param in a separate line else. Other opinions?
Please verify if this does what I think it does and whether you like it.

File: native/.idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -19,6 +19,8 @@
       <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
       <option name="ALIGN_MULTILINE_FOR" value="false" />
       <option name="SPACE_BEFORE_METHOD_PARENTHESES" value="true" />
+      <option name="CALL_PARAMETERS_WRAP" value="1" />
+      <option name="METHOD_PARAMETERS_WRAP" value="1" />
```
Comment (klinzo) [inline native/.idea/codeStyles/Project.xml:23]: I think the option we want is chop down if to long:
```suggestion
      <option name="CALL_PARAMETERS_WRAP" value="5" />
      <option name="METHOD_PARAMETERS_WRAP" value="5" />
```
Please verify if this does what I think it does and whether you like it.

**Review by klinzo — APPROVED** [review]

**Comment by klinzo** [comment]
I think the code style should be changed for web/project wide as well.

**Review by maxammann — APPROVED** [review]

**Comment by maxammann** [comment]
Try to reapply the JS Standard tyle:

image

**Comment by klinzo** [comment]
@maxammann actually I think this is not what we want here and changes nothing. Lets discuss this in the teamcall today.

**Comment by maxammann** [comment]
It changes one setting :D I tried already

**Comment by SteffiStuffel** [comment]
Adjusted code style for native and web. Tested several files and most of them haven't canged by auto-formatting.

**Review by klinzo — APPROVED** [review]

**Pull request merged**